src/index.scss

```diff
@@ -1,3 +1,763 @@
-$enable-shadows: true;
-$enable-gradients: true;
-@import "~bootstrap/scss/bootstrap";
\ No newline at end of file
+// Breakpoints
+
+$MD: 768px;
+
+// Culori
+
+/////////////////////////////////Light Mode/////////////////////////////////
+/// Backgrounds
+$primary-color: #FFFFFF;
+$secondary-color: #E2E4E9;
+$third-color: #D3D3D3;
+$fourth-color: #bfc2c9;
+/// Text color
+$text-color: #363C49;
+$clickable-color: #2D6A9F;
+/// Navbar
+$button-color: #f1faee;
+$button-color-hover: #457b9d;
+
+/////////////////////////////////Dark Mode/////////////////////////////////
+/// Backgrounds
+$primary-color-dark: #0B0C0F;
+$secondary-color-dark: #2B303B;
+$third-color-dark: #656464;
+$fourth-color-dark: #4b5368;
+/// Text color
+$text-color-dark: #ffffff;
+$clickable-color-dark: #f3c3c3;
+/// Navbar
+$button-color-dark: #2d2476;
+$button-color-hover-dark: #673d8d;
+
+// Întreaga pagină
+
+html {
+    background-color: $primary-color;
+    color: $text-color;
+    font-family: 'Roboto Condensed';
+    @media (prefers-color-scheme: dark) {
+        background-color: $primary-color-dark;
+        color: $text-color-dark;
+    }
+    font-size: 1.1em;
+
+}
+
+.thebody {
```

Comment:
Dacă folosești denumiri de clase formate din două sau mai multe cuvinte, aș recomanda să le separi cumva. De exemplu `.the-body`.